### PR TITLE
Allow secborgs to use the L6 module

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
title explain it all

## Why / Balance
the module can only be obtained as rare expedition loot or from looting a fallen syndie borg, i feel like our secborgs should be able to receive said upgrade on rare occasion and it shouldnt lead to too many issues
## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Avalon-Proto** [comment]
Ultra rare item that gives a borg a mega power boost, sounds like fun. Just hope its not emagged

**Comment by deltanedas** [comment]
infinite L6 for the shadow factory yummy

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml [hunk 1]
```diff
@@ -618,6 +618,9 @@
         - PinpointerSyndicateNuclear
     - type: BorgModuleIcon
       icon: { sprite: Interface/Actions/actions_borg.rsi, state: syndicate-l6c-module }
+    - type: Tag
+      tags:
+        - BorgModuleSecurity # DeltaV allows secborgs to use
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml]: comment on first line

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by therealDLondon** [comment]
Im going to be real.
Im very much on the page of denying this one, I dont like  the idea of secborgs with L6s, especially with the laws being the way they are rn

**Pull request closed without merging**